tests/test_framework/phpbb_test_case.php

```diff
@@ -27,6 +27,7 @@ public function __construct($name = NULL, array $data = array(), $dataName = '')
 			'PHP_Token_Stream_CachingFactory' => array('cache'),
 
 			'phpbb_database_test_case' => array('already_connected', 'last_post_timestamp'),
+			'Prophecy\Doubler\NameGenerator' => array('counter'),
```

Comment:
Travis errors might also come from here, as the errors are related to "duplicating"...